.github/workflows/basic.yml

```diff
@@ -28,6 +28,13 @@ jobs:
       - name: Install protoc
         run: |
           sudo apt-get update && sudo apt-get install -y protobuf-compiler
+      
+      - name: Install TDX dependencies
+        run: |
+          sudo curl -L https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+          sudo echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+          sudo apt-get update
+          sudo apt-get install -y libtdx-attest-dev
 
       - name: Build and install with default features
```

Comment:
hmmm -- don't we need a new `.github/workflows/cc_kbc.yml` to build with `make KBC=cc_kbc` ?

I'm not convinced the current CI check on this PR actually compiles the code of the TDX attester.

Reply:
`cc_kbc.yml` is a good idea.